api/src/pcapi/routes/backoffice/finance/validation.py

```diff
@@ -67,8 +67,13 @@ def check_total_amount(
 ) -> bool:
     # Temporary: The total_amount field is optional only for multiple bookings incident (no need for total overpayment)
     if not input_amount:
-        flash("Le montant de l'incident est vide.", "warning")
+        flash("Impossible de créer un incident d'un montant de 0€.", "warning")
         return False
+
+    if input_amount < 0:
+        flash("Le montant d'un incident ne peut être négatif.", "warning")
+        return False
+
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Quid du cas des gestes commerciaux ?

Reply:
yes